app/controllers/users_controller.rb

```diff
@@ -0,0 +1,49 @@
+class UsersController < ApplicationController
+
+  get '/signup' do
+      if Helper.logged_in?(session)
+        redirect to "/tweets"
+      else
+        erb :'users/create_user'
+      end
+    end
+
+  post '/signup' do
+    if params[:username].empty? || params[:password].empty? || params[:email].empty?
```

Comment:
I think you could use [ActiveRecord validations](http://guides.rubyonrails.org/active_record_validations.html) (e.g. `validates_presence_of :username`) to perform these types of checks more easily.